easybuild/easyconfigs/z/zlib/zlib-1.2.8-GCC-5.1.0-binutils-2.25.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.8'
+
+homepage = 'http://www.zlib.net/'
+description = """zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
+ not covered by any patents -- lossless data-compression library for use on virtually any
+ computer hardware and operating system."""
+
+toolchain = {'name': 'GCC', 'version': '5.1.0-binutils-2.25'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/libpng/files/zlib/%(version)s', 'download')]
```

Comment:
well, this is an auto-resolving mirror, which is official (it's also mentioned on zlib.net)

that, and I don't want to make this change in all zlib easyconfigs... ;-)